contents/blog/leo-anthias-interview.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2021-07-28
+title: Towards Open Source Data Science - An interview with Datapane CEO, Leo Anthias
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: mo-shehu
+featuredImage: ../images/blog/leo-anthias-interview.png
+---
+
+Unlike most tech founders, Leo Anthias didn’t study computer science at university. Instead, he took Persian History and Modern American Drama after a chance encounter with a professor who made medieval history fascinating. Despite his non-tech academic background, however, Leo has been coding for years, from early roles writing Perl scripts for freelance clients to his current job as CEO and co-founder of Datapane.
```

Comment:
A couple of general questions:

1. Should we add a pre-amble about this covering that we're chatting to people who are part of the PostHog community? "This is the first in a series of..." type of thing?
2. Are we getting the people we interviewed to review before we publish? Or do we publish, drop them a link and ask them to check, and then socialize?

Reply:
1. Will add a preamble.
2. They'll get to review before we go live.